Wikipedia/Code/DescriptionEditViewController.swift

```diff
@@ -95,7 +170,7 @@ protocol DescriptionEditViewControllerDelegate: AnyObject {
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
-        descriptionTextView.becomeFirstResponder()
+        self.descriptionTextView.becomeFirstResponder()
```

Comment:
I suggest removing this `becomeFirstResponder()` line entirely (which focuses the textfield) and replacing it with `showTempAccountToast`, to display the toast at the bottom. If we kept the first responder focus the keyboard would hide the toast.